googletest/include/gtest/internal/gtest-type-util.h

```diff
@@ -67,6 +67,22 @@ inline std::string CanonicalizeForStdLibVersioning(std::string s) {
       s.erase(strlen("std"), end - strlen("std"));
     }
   }
+
+  /* Strip redundant spaces in typename to match MSVC */
+  /* For example, std::pair<int, bool> -> std::pair<int,bool> */
+  const std::string &toSearch = ", ";
```

Comment:
Do not rely on lifetime extension if it's not necessary.
In this case, `const char to_search[] = ", ";` would be preferred.

Also, please use snake_case instead of camelCase for local variables in accordance with Google's style guide.